### PR TITLE
support searching by cluster alias

### PR DESCRIPTION
**Pull request by MaxFedotov**
Related issue: #935 

Description:
Support for search using cluster alias. Show cluster alias instead of cluster name in search results UI if it is set

### CODE REVIEW

**Comment by shlomi-noach** [comment]
Thank you! Since this is a web UI enhancement, can you please paste screenshots for "before" and "after" this change?

File: resources/public/js/orchestrator.js [hunk 1]
```diff
@@ -933,7 +933,11 @@ function renderInstanceElement(popoverElement, instance, renderType) {
       contentHtml += '<p><strong>Master</strong></p>';
     }
     if (renderType == "search") {
-      contentHtml += '<p>' + 'Cluster: <a href="' + appUrl('/web/cluster/' + instance.ClusterName) + '">' + instance.ClusterName + '</a>' + '</p>';
+      if (instance.SuggestedClusterAlias) {
+        contentHtml += '<p>' + 'Cluster: <a href="' + appUrl('/web/cluster/' + instance.SuggestedClusterAlias) + '">' + instance.SuggestedClusterAlias + '</a>' + '</p>';
```
Comment (shlomi-noach) [inline resources/public/js/orchestrator.js]: While the existing code is correct, it is more explicit to strictly use `/web/cluster/alias`.
```suggestion
        contentHtml += '<p>' + 'Cluster: <a href="' + appUrl('/web/cluster/alias/' + instance.SuggestedClusterAlias) + '">' + instance.SuggestedClusterAlias + '</a>' + '</p>';
```

**Review by shlomi-noach — APPROVED** [review]
Looks good! I made a code suggestion that I think should go in. Looking forward to screenshots.

**Comment by MaxFedotov** [comment]
Hello Shlomi,
Thanks for review.
Here is screenshot before:
Screen Shot 2019-08-02 at 16 01 56
And after:
Screen Shot 2019-08-02 at 16 02 43

**Pull request merged**